### PR TITLE
Add Cloud Scale Inc as pro services

### PR DESCRIPTION
**Pull request by nandita-cloudscaleinc**
Please provide a description for what this PR is for.

Add Cloud Scale® Inc as pro service provider for ISTIO service mesh

And to help us figure out who should review this PR, please put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by nandita-cloudscaleinc** [comment]
Can someone pls review this. Thanks !

**Comment by craigbox** [comment]
we need to close out https://github.com/istio/istio.io/issues/11240 first I think.

I would also recommend you remove all the "® Inc" (or only include it once) please.

**Comment by nandita-cloudscaleinc** [comment]
> we need to close out #11240 first I think.
> 
> I would also recommend you remove all the "® Inc" (or only include it once) please.

Thanks Craig. Addressed the comments and kept it only at the required places!

**Comment by nandita-cloudscaleinc** [comment]
@ericvn can you pls review this request

File: data/companies.yml [hunk 1]
```diff
@@ -292,6 +292,13 @@ pro_services:
...
+  - name: "Cloud Scale® Inc"
+    logo: "/logos/cloudscaleinc.png"
+    url: "https://cloudscaleinc.com/"
+    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, Identify gaps in technology & skills, and prepare a success plan – The ones that will help you accomplish your business objectives"
+    details:
+      - "Cloud Scale platform and service mesh experts puts distribution & management in the palm of your hands. From productivity to continuity, from security to remote monitoring & management – our growing portfolio of services has got you covered. "
+      - "Cloud consulting and Istio service mesh experts by Cloud Scale helps in solving all of the needs by selecting the right platforms, cloud solutions and platform needs that integrate smoothly with legacy & next-generation technology’s and deliver optimal return on technology investment (TCO)"
```
[the remark this answers is not in the document]
Comment (nandita-cloudscaleinc) [reply data/companies.yml]: addressed the review comments. Thanks for the feedback

**Comment by nandita-cloudscaleinc** [comment]
Can this PR be pls reviewed . Thanks!

cc - @ericvn @craigbox

**Comment by nandita-cloudscaleinc** [comment]
Gentle Reminder. Can someone pls review this. Thanks

File: data/companies.yml [hunk 2]
```diff
@@ -292,6 +292,13 @@ pro_services:
       - "InfraCloud is your end-to-end Istio service mesh consulting & implementation partner - whether you're just starting out or want to take it to the next level."
       - "The team of service mesh consultants assesses your microservice landscape and proposes the right strategy and roadmap for your goals and deploy it successfully. Plus, get hands-on experience and training with InfraCloud team by your side enabling you to make the most of your Istio deployment."
       - "InfraCloud also offers enterprise-grade Istio service mesh experts, whenever and wherever you need them to resolve your production issues. The service mesh experts can help you optimize, migrate to Istio or build custom features for your specific needs."
+  - name: "Cloud Scale® Inc"
+    logo: "/logos/cloudscaleinc.png"
+    url: "https://cloudscaleinc.com/"
+    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, identify gaps in technology & skills, and prepare a success plan – The ones that will help you accomplish your business objectives."
```
Comment (craigbox) [inline data/companies.yml:298]:
```suggestion
    description: "Cloud Scale® Inc experts assess your current applications, runtime deployments, service maps and use-cases, find the right mix of cloud platforms for your workloads, identify gaps in technology & skills, and prepare a success plan."
```

I think it's too much longer than the others.  If this LGTM, I can merge.

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by nandita-cloudscaleinc** [comment]
addressed review comments. Please let me know if there's any other feedback!

**Review by craigbox — APPROVED** [review]

**Pull request merged**